website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,68 @@
+
+## New Module Support!
+
+We've begun addressing the Windows community module support gap - starting with two contributions from us! The [**WebView**](https://github.com/react-native-community/react-native-webview) and [**Camera**](https://github.com/react-native-community/react-native-camera) modules.
+
+On top of those initial two, we're looking at a whole bunch of other ones we know are awesome and essential to have when building a React Native app in general. Check out the full list of what we're tracking and planning to tackle next on our [GitHub Project Board](https://github.com/microsoft/react-native-windows/projects/23) for modules.
+
+We're also introducing release stages!
+
+### WebView
+
+![react-native-webview-module](assets/final_demo_gif.gif)
+
+The WebView module sets you up with the barebones web hosting tech that's available nativly on the device you're targetting - as a bonus it works with macOS as well!
+
+### Camera
+
+![react-native-camera-module](assets/react-camera.png)
+
+The Camera module is another big step towards making it easier to give you unfettered access to native APIs.
+
+Just like on mobile, when you define a CameraFeed module, and the device you're running the app on has a camera attached, the component will hookup to the correct native APIs to give you access to the hardware needed to snap sweet pics.
+
+## React Native on the Surface Duo
+
+![react-native-duo-emu](https://devblogs.microsoft.com/surface-duo/wp-content/uploads/sites/53/2020/04/reactnative-emulator-800.png)
+
+React Native runs on the Surface Duo!
+
+This is a little bit to be expected, as React Native works great today with Android, but just running on the new Surface Duo device isn't enough. We've also added support for awareness of the dual-screen device and it's status through shipping two important APIs: **TwoPaneView** and **DeviceInfoModule**.
+
+To learn more about developing on the Surface Duo with React Native, [go check out our latest blog post](https://devblogs.microsoft.com/surface-duo/build-react-native-apps-for-microsoft-surface-duo/) detailing the experience.
+
+## New Release Processes
+
+We are also rolling out a new release process and cadence in order to better ensure release quality as well as give consumers of **React Native for Windows** earlier access to the new features if needed.
+
+| <div style="width:120px">Title</div> | <div style="width:300px">Expectations</div> | <div style="width:300px">Promises</div> |
+|:--|:-----|:-----|
+| **Canary** | Released per day (ish) or per commit, depending on what's least expensive and necessary. All CI's pass.| The release builds, maybe it only builds on one flavor, but CI's work and pass.<br>No white glove experience, none to minimal communication about the release with full knowledge to consumers that things could be in a bad state/broken in places.|

```

Comment:
This doesn't quite match the current naming that happened organically. "Official Release" has corresponded to the npm dist-tag of `latest` out of tooling expectation. Preview is the same. Nightly(ish) builds are currently tagged as `master`, but `canary` came up  as a possible name as well (we can change things if we want to).

Canary/Master aren't guaranteed to be tied to a released version of upstream React Native, which ends up being big for consumers since they can't necessarily use a bunch of the things they're used to. 

CI should be passing for all builds, regardless of channel.

Preview builds aligning to FB stable builds when we're caught up means we might not have done much manual validation of the initial preview build, or it may have some issues.